### PR TITLE
Add effect for Sun's Fury

### PR DESCRIPTION
**Pull request by simonward**

### CODE REVIEW

**Comment by simonward** [comment]
Added holy trait instead of 1 extra spirit damage.

File: packs/spells/suns-fury.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
         "damage": {},
         "defense": null,
         "description": {
-            "value": "<p>The target weapon becomes wreathed in a glowing flame. The weapon deals an additional @Damage[1d4[fire]] damage and @Damage[1[spirit]] damage on a successful Strike. In addition, the flame causes the weapon to glow as bright as a torch. You can @UUID[Compendium.pf2e.actionspf2e.Item.Dismiss] the spell.</p>"
+            "value": "<p>The target weapon becomes wreathed in a glowing flame. The weapon gains the holy trait and deals an additional 1d4 fire damage on a successful Strike. In addition, the flame causes the weapon to glow as bright as a torch. You can Dismiss the spell.</p>\n<p>@UUID[Compendium.pf2e.spell-effects.Item.Effect: Sun's Fury]</p>"
```
Comment (TikaelSol) [inline packs/spells/suns-fury.json:14]: I don't like us doing this much editorializing. Paizo gave us guidelines on NPCs, but not for PC spells.

Comment (simonward) [reply packs/spells/suns-fury.json:14]: Fair enough. What about adding the holy trait?

Comment (TikaelSol) [reply packs/spells/suns-fury.json:14]: I think leave it until we have errata.

**Pull request merged**